addons/barcodes/__openerp__.py

```diff
@@ -1,23 +1,29 @@
 {
     'name': 'Barcodes',
-    'version': '1.0',
-    'category': '',
-    'sequence': 6,
-    'summary': 'Barcode Nomenclatures Setup',
+    'version': '2.0',
+    'category': 'Hidden/Dependency',
+    'summary': 'Barcodes Scanning and Parsing',
     'description': """
+This module adds support for barcode scanning and parsing.
 
-=======================
+Scanning
+--------
+Use an USB scanner (that mimics keyboard inputs) in order to work with barcodes in Odoo.
+The scanner must be configured to use no prefix and a carriage return or tab as suffix.
+The delay between each character input must be less than or equal to 50 milliseconds.
+This is how most barcode scanners will work out of the box.
+However, make sure the scanner uses the same keyboard layout as the device it's plugged in.
```

Comment:
What's important here, definitely for the card reader device is that the keyboard layout on the client is set to exactly US QWERTY (not US QWERTY International or something like that, because that will cause issues). So I suggest we say that **unless** there are devices that require something else. Do you have devices that require eg. French AZERTY? I actually hope not, because that could cause issues.